integreat_cms/cms/utils/mfa_utils.py

```diff
@@ -24,3 +24,20 @@ def generate_challenge(challenge_len):
             for i in range(challenge_len)
         ]
     )
+
+
+def get_passwordless_mfa_user_id(request):
+    """
+    Get the user id from the session. This method can be used for the passwordless login, as well as the usual login mode.
+
+    :param request: The current request
+    :type request: ~django.http.HttpRequest
+
+    :return: The user id
+    :rtype: int
+    """
+    if "mfa_user_id" in request.session:
+        return request.session["mfa_user_id"]
+    if "mfa_passwordless_user_id" in request.session:
+        return request.session["mfa_passwordless_user_id"]
+    return None
```

Comment:
If we don't differentiate between the two session variables, do we really need both? Or could we just use `mfa_user_id` for all cases?
Also, why don't we also return the user object if the id exists? I mean this will be the next step anyway, whenever this function is called. So I guess it would also make sense to call this `get_mfa_user()` and return either a user object or `None`?

Reply:
If I remember correctly, Jonas could think of a situation where attackers could mitigate the 2FA, if they somehow fake the passwordless id and use this function for a 2FA authentication, even if the user didn't activated it.

However, I applied your changes but kept the two separate IDs.

Reply:
If this vulnerability exists, then your current code does not prevent it.
`get_mfa_user()` returns the user of the `mfa_passwordless_user_id` session variable even if it's called from the totp view and vice versa. If this is not a problem, we can use the same session variable, and if it is a problem, we have to use two different methods.

Reply:
I removed the extra ID and only use the mfa_user_id.